### PR TITLE
gitignore and clean repos.tar.gz

### PR DESCRIPTION
**Pull request by anna-zzz**

### CODE REVIEW

**Comment by sjrd** [comment]
A better fix would be to put that file inside `target/`.

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,8 @@
 *.class
 *.log
 
+repos.tar.gz
```
Comment (olafurpg) [inline .gitignore]: 👍

File: build.sbt [hunk 2]
```diff
@@ -49,6 +49,7 @@ lazy val commonSettings = Seq(
   assemblyJarName in assembly := "scalafmt.jar",
   ScoverageSbtPlugin.ScoverageKeys.coverageExcludedPackages :=
       ".*Debug;.*ScalaFmtLogger;.*Versions",
+  cleanFiles <+= baseDirectory(_ / "repos.tar.gz"),
```
Comment (olafurpg) [inline build.sbt]: Is this necessary? I would prefer not to re-download it on every sbt clean.

**Comment by olafurpg** [comment]
Good catch, seems like I ignore `*.gz` in my `$HOME/.gitignore`.

The file is not inside `target/` because I want to keep it between `sbt clean`s. Is that a terrible idea? As long as I don't change the contents of `repos.tar.gz` I think it will be fine.

**Comment by sjrd** [comment]
> Is that a terrible idea?

No, it's reasonable.

File: build.sbt (hunk 2 above)
Comment (anna-zzz) [inline build.sbt]: Nope, but it's a PR open for discussion :) Easier to add a line and submit PR than to ask. Will remove.

**Comment by olafurpg** [comment]
LGTM, thanks! :)

**Pull request merged**